Source/DataSources/CorridorGeometryUpdater.js

```diff
@@ -165,10 +165,10 @@ define([
     };
 
     CorridorGeometryUpdater.prototype._isOnTerrain = function(entity, corridor) {
-        var isColorMaterial = this._materialProperty instanceof ColorMaterialProperty;
+        //var isColorMaterial = this._materialProperty instanceof ColorMaterialProperty;
```

Comment:
Just a reminder here, below, and throughout to not leave any commented out code in this PR.